src/main/resources/logback.xml

```diff
@@ -1,19 +1,18 @@
 <configuration>
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <filter class="stream.flarebot.flarebot.util.errorhandling.Filter" />
         <file>./latest.log</file>
         <encoder>
             <pattern>[%d{yyyy.MM.dd HH:mm}] [%t] [%level] %msg%n</pattern>
         </encoder>
     </appender>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <filter class="stream.flarebot.flarebot.util.errorhandling.ErrorCatcher" />
-        <filter class="stream.flarebot.flarebot.util.errorhandling.Filter" />
```

Comment:
Why did you remove the filter?

Reply:
Because the same functionality exists in error catcher, would be a redundant filter